lib/ProductOpener/Ingredients.pm

```diff
@@ -1186,6 +1186,7 @@ sub parse_ingredients_text($) {
 					}
 				}
 
+				# Check ingredient against labels taxonomy. If it exists there, don't parse it as an ingredient, and add it to labels. See also further down.
```

Comment:
It's a bit more complex: it checks if it is an ingredient + a label, for a limited subset of labels:
my @labels = ("en:fair-trade-organic", "en:organic", "en:fair-trade", "en:pgi");

e.g. "organic tomatoes" -> we add "en:organic" to labels, and we keep "tomatoes" as the ingredient

```suggestion
				# Check if we have an ingredient + some specific labels like organic and fair-trade.
				# If we do, remove the label from the ingredient and add the label to labels
```